src/traits/window.rs

```diff
@@ -6,8 +6,21 @@ use glib::translate::{from_glib_none, ToGlibPtr};
 use ffi;
 use glib::to_gboolean;
 use cast::GTK_WINDOW;
+use gdk;
 
 pub trait WindowTrait : ::WidgetTrait {
+    fn move_(&self, x: i32, y: i32) {
```

Comment:
Not at all. Otherwise, why uint32_t and similar types would have been created ? Some links :
 * http://stackoverflow.com/questions/2533319/c-variable-type-sizes-are-machine-dependent-is-it-really-true-signed-unsig
 * http://www.nongnu.org/c-prog-book/online/x401.html
 * http://www.linuxquestions.org/questions/programming-9/int-size-is-compiler-dependent-or-os-dependent-915923/